osgp/shared/shared/src/test/java/org/opensmartgridplatform/shared/mappers/XMLGregorianCalendarToInstantConverterTest.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright 2020 Alliander N.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.shared.mappers;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.util.GregorianCalendar;
+
+import javax.xml.datatype.DatatypeConfigurationException;
+import javax.xml.datatype.DatatypeFactory;
+import javax.xml.datatype.XMLGregorianCalendar;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import ma.glasnost.orika.MapperFacade;
+import ma.glasnost.orika.MapperFactory;
+import ma.glasnost.orika.impl.DefaultMapperFactory;
+
+class XMLGregorianCalendarToInstantConverterTest {
+
+    public static final ZoneId UTC = ZoneId.of("UTC");
+
+    private final MapperFactory mapperFactory = new DefaultMapperFactory.Builder().build();
+    private MapperFacade mapper;
+
+    @BeforeEach
+    void before() {
+        this.mapperFactory.getConverterFactory().registerConverter(new XMLGregorianCalendarToInstantConverter());
+        this.mapper = this.mapperFactory.getMapperFacade();
+    }
+
+    @Test
+    void testMappingToInstantFromXmlGregorianCalenderWithTimeZone() throws DatatypeConfigurationException {
```

Comment:
Another pet peeve of mine. In my opinion naming test methods starting with "test" was useful when JUnit needed this to know which methods to execute as part of the test framework execution, a convention made obsolete by the `Test` annotation. Four characters that can be put to better use!
Just hoping to grow the community against test name duplication, if you persist in liking this naming and annotating pattern and be old school mixed with something relatively new, be my guest. ;)

Reply:
I've deactivated sonar rule S3578 on my local development environment and renamed the methods.